bundle/edu.gemini.itc/src/main/scala/edu/gemini/itc/base/DatFile.scala

```diff
@@ -114,8 +112,8 @@ object DatFile {
           l +=(Grating(name, resolvingPower, blaze, -999999, dDis, resolution))
         }
         case e : Exception => {
-
-          Log.severe("NOT ANALYZED THE EXCEPTION, PLEASE CONTACT WITH A PROGRAMMER. LINE 125 OF THE DatFile.scala")
+          Log.log(Level.SEVERE, "The grating configuration is not correct. ", e)
+          throw e
```

Comment:
This change is fine, :+1:, but just want to note that I don't think even an incorrect configuration will ever trigger this. If `tmp` isn't a parseable as a `Double` then the `NumberFormatException` is caught above and if there is a problem handling it (say in `arrays.apply(tmp)`, it won't be caught here.  We'll see this exception in the case of memory issues and problems that can happen anywhere else.

Reply:
Perhaps the `Grating` constructor can also throw exceptions though in which case this log would also be relevant.